RecoLocalCalo/HGCalRecProducers/python/HGCalUncalibRecHit_cfi.py

```diff
@@ -41,7 +41,8 @@
     HGCHEBConfig = cms.PSet(
         isSiFE  = cms.bool(False),
         adcNbits      = hgchebackDigitizer.digiCfg.feCfg.adcNbits,
-        adcSaturation = hgchebackDigitizer.digiCfg.feCfg.adcSaturation_fC
+        adcSaturation = hgchebackDigitizer.digiCfg.feCfg.adcSaturation_fC,
+        fCPerMIP      = cms.vdouble(1.0,1.0,1.0) #dummy values, it's scintillator
```

Comment:
The change for reading the code is a few months back, back when the BH was being dealt with through the Hcal-only paths. Since the old BH config was never used it wasn't updated until now.